### PR TITLE
Discussion v1

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Adds @guardian/discussion-rendering@1.0.0

## Why?
Ready for internal testing

### CODE REVIEW

File: docs/contributing/detailed-setup-guide.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 <!-- Automatically created with yarn run createtoc and on push hook -->
 
-- [Chat](#chat)
+- [High level diagram](#high-level-diagram)
```
Comment (shtukas) [inline docs/contributing/detailed-setup-guide.md:7]: Thanks!

Comment (oliverlloyd) [reply docs/contributing/detailed-setup-guide.md:7]: You're welcome. Although to be perfectly honest I have no idea how this appeared here

Comment (oliverlloyd) [reply docs/contributing/detailed-setup-guide.md:7]: I'm guessing this: `Automatically created with yarn run createtoc` did it

Comment (gtrufitt) [reply docs/contributing/detailed-setup-guide.md:7]: Yup - may be that @shtukas's hooks are not running correctly?

**Pull request merged**